HPXMLtoOpenStudio/resources/EPvalidator.xml

```diff
@@ -96,17 +96,32 @@
       <sch:assert role='ERROR' test='count(h:MiscLoads/h:FuelLoad[h:FuelLoadType[text()="fireplace"]]) &lt;= 1'>Expected 0 or 1 element(s) for xpath: MiscLoads/FuelLoad[FuelLoadType[text()="fireplace"]]</sch:assert> <!-- See [FuelLoad] -->
+      <!-- Warnings -->
+      <sch:report role='WARN' test='count(h:Appliances/h:ClothesWasher) = 0'>No clothes washer specified, the model will not include clothes washer energy use.</sch:report>
+      <sch:report role='WARN' test='count(h:Appliances/h:ClothesDryer) = 0'>No clothes dryer specified, the model will not include clothes dryer energy use.</sch:report>
+      <sch:report role='WARN' test='count(h:Appliances/h:Dishwasher) = 0'>No dishwasher specified, the model will not include dishwasher energy use.</sch:report>
+      <sch:report role='WARN' test='count(h:Appliances/h:Refrigerator) = 0'>No refrigerator specified, the model will not include refrigerator energy use.</sch:report>
+      <sch:report role='WARN' test='count(h:Appliances/h:CookingRange) = 0'>No cooking range specified, the model will not include cooking range/oven energy use.</sch:report>
+      <sch:report role='WARN' test='count(h:Systems/h:WaterHeating/h:WaterHeatingSystem) = 0'>No water heater specified, the model will not include water heating energy use.</sch:report>
+      <sch:report role='WARN' test='count(h:MiscLoads/h:PlugLoad[h:PlugLoadType[text()="other"]]) = 0'>No 'other' plug loads specified, the model will not include misc plug load energy use.</sch:report>
+      <sch:report role='WARN' test='count(h:MiscLoads/h:PlugLoad[h:PlugLoadType[text()="TV other"]]) = 0'>No 'TV other' plug loads specified, the model will not include television plug load energy use.</sch:report>
     </sch:rule>
   </sch:pattern>  
   
   <sch:pattern name='[Site]'>
     <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:BuildingSummary/h:Site'>
-      <sch:assert role='ERROR' test='not(h:SiteType) or count(h:SiteType[text()="urban" or text()="suburban" or text()="rural"]) = 1'>Expected 1 element(s) for xpath: [not(SiteType)] | SiteType[text()="urban" or text()="suburban" or text()="rural"]</sch:assert>
+      <sch:assert role='ERROR' test='count(h:SiteType) &lt;= 1'>Expected 0 or 1 element(s) for xpath: SiteType</sch:assert> <!-- See [SiteType] -->
       <sch:assert role='ERROR' test='count(h:extension/h:ShelterCoefficient) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/ShelterCoefficient</sch:assert>
       <sch:assert role='ERROR' test='count(h:extension/h:Neighbors) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/Neighbors</sch:assert> <!-- See [Neighbors] -->
     </sch:rule>
   </sch:pattern>
 
+  <sch:pattern name='[SiteType]'>
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:BuildingSummary/h:Site/h:SiteType'>
+      <sch:assert role='ERROR' test='contains("_urban_ _suburban_ _rural_", concat("_", text(), "_"))'>Expected "text()" for xpath: "urban", "suburban", or "rural"</sch:assert>
```

Comment:
In https://github.com/NREL/OpenStudio-HPXML/pull/489/files, you are proposing messages like "Expected the value to be greater than or equal to 0". It'd be nice if we were more consistent here. How about we change these warnings from:
Expected "text()" for xpath: ...
to:
Expected value to be: ...